libs/core/src/lib/message-strip/message-strip.component.html

```diff
@@ -1,3 +1,13 @@
+<span class="fd-accessible-hidden-message" [attr.aria-label]="_getMessage()"></span>
```

Comment:
I do not think this invisible element is necessary, this aria-label can be placed on the <fd-message-strip> host element

Reply:
have followed your approach and tested with chrome screen reader. Thank you!
Will get in touch with a11y member to test on JAWS as well. Seems like voiceover is not a good way to test screen reader stuff!